### ISSUE TITLE
Splash page UNOPS Idai description 

### ISSUE THREAD

**Issue by akahure** [onaio/gisida#446]
In UNOPS Idai recover, the client has presented us with detailed logical framework that they would like to have added to Gisida. The request is initially show a summary view of the progress towards the main **Project Outcome results** with the sub outcome results, activities, and budget tied to these activities. 

Logframe 

<img width="1086" alt="Screenshot 2020-03-24 at 17 07 14" src="https://user-images.githubusercontent.com/43808025/77434447-f00ab180-6df1-11ea-82f9-ba63c8a97af3.png">

Results Doc - [ZIRP Results Matrix - Visualization (Revised).docx](https://github.com/onaio/gisida/files/4375738/ZIRP.Results.Matrix.-.Visualization.Revised.docx)

From the image above, the objective will be (1) to put together a design for the client's approval on how we can show all Immediate outcome results: 
- Overall progress towards the performance for the Min targets vs the Actual (data coming from Ona)
- Overall budget expenditure for the indicators and corresponding activities (data coming from Ona)

One possible visualization could be a modification of what we currently have on the splash page for Gisida sites, where we show a bar with the targets and the actual values reported additional bars for the sub-indicators. 

![Screenshot - 2020-03-24T180912 339](https://user-images.githubusercontent.com/43808025/77441775-dcb01400-6dfa-11ea-81d3-d1a74bb28119.png)

We can include another bar possibly to show budget expenditure with a legend to distinguish the two? 
example

![Screenshot - 2020-03-24T172350 700](https://user-images.githubusercontent.com/43808025/77436216-45e05900-6df4-11ea-8ed8-1fa2fa9e82a5.png)

source: https://idairecovery-zimbabwe.onalabs.org/

(2) Allow for users to click on the main indicator results and view: 
- The project outcome results and progress for the actual values in relation to the targets. All this data will come from an Ona form

**Comment by rowo** [comment]
Update on the UNOPS splash thing to get a conversation with the client going. First screen is that first ask, showing all project outcome results in one screen. If you click on one card’s title, you get to the second ask, showing project outcome results + activities + budget altogether.  

![summary splash](https://user-images.githubusercontent.com/1584163/77668040-4ba37f00-6f59-11ea-9964-e7ef2ccea1ab.png)
![outcome-emergency livelihood](https://user-images.githubusercontent.com/1584163/77668043-4cd4ac00-6f59-11ea-9b5e-0164116172e6.png)

**Comment by akahure** [comment]
@rowo I checked with the client on the review meeting and they requested that we make the updates to the site first, with the data they shared for the targets and actuals. I think we should probably go ahead with this. Kindly let me know if there is a specific way you would like the data and I can put it together. Maybe in a CSV, Doc 
On chat, you had mentioned we should probably do it via the existing website using HTML.
@engwiri

**Comment by engwiri** [comment]
@akahure this will create a lot of unnecessary back and forth. We need to make sure sign off happens at each stage and we are not going through endless prototypes which we have to build each time... but if @rowo is ok with building the prototypes as they are then that's ok

**Comment by akahure** [comment]
> @akahure this will create a lot of unnecessary back and forth. We need to make sure sign off happens at each stage and we are not going through endless prototypes which we have to build each time... but if @rowo is ok with building the prototypes as they are then that's ok

I agree, sign-offs are important, he client approved the designs, but they requested that we include the actual values for the indicators in the mockup before the splash page signoff can be given from Djibrilla despite my attempts to get Sheila and him on a call.

**Comment by rowo** [comment]
@akahure just want to make sure you are not waiting on me for anything. Just waiting on a spreadsheet from you with the numbers in an easy to copy format as per our discussion on Slack. Maybe share it here in this issue?

**Comment by akahure** [comment]
@rowo I have completed the doc as discussed on chat. 
You can find the Main outcome and sub outcome indicators here https://docs.google.com/spreadsheets/d/19S1R6H8cfW8WzcXDcxgxzc3cy-OU8lMQu5HEpW6Vl_k/edit?usp=sharing

Two things to note
1. In situations where the Actual is higher than the target, they would like the progress bar show differently, from the current one showing a simple black progress bar. Perhaps we can use a green color to indicate it has exceeded. 
![Screenshot - 2020-04-09T145808 586](https://user-images.githubusercontent.com/43808025/78892729-8f76a800-7a72-11ea-9d74-3f35174091a0.png)
2. Some Outcome results only have one target and not two, as is the case with most of them with a Max and a Min. The progress towards both should be highlighted i.e. Actual vs Min and Max.

**Comment by rowo** [comment]
@akahure For the second point.. can you add a column to the data with the percent towards max?

**Comment by rowo** [comment]
@akahure and just to confirm, this is what they mean?

<img width="434" alt="Screen Shot 2020-04-13 at 10 16 06 AM" src="https://user-images.githubusercontent.com/1584163/79127409-d7961300-7d6f-11ea-946c-d47a0f1064cd.png">

**Comment by akahure** [comment]
> @akahure For the second point.. can you add a column to the data with the percent towards max?

okay cool will do

**Comment by akahure** [comment]
@rowo I made updates to the doc here https://docs.google.com/spreadsheets/d/19S1R6H8cfW8WzcXDcxgxzc3cy-OU8lMQu5HEpW6Vl_k/edit

**Comment by alepietrobon** [comment]
hey @rowo , just an idea after some convo with Alex this morning. To report those indicators that have a min an max value, have we considered just a single line showcasing both the range and the actual?  
If any % results needs to be reported, we can confirm with the client if they need % of minimum OR % of median target. That might depend a bit on context.

![image](https://user-images.githubusercontent.com/8207502/79843409-85ca3a00-83ba-11ea-9571-8f9c1834a649.png)

Thoughts?

**Comment by rowo** [comment]
@alepietrobon Thanks for pointing that out. I think that is a pleasantly compact way to display the data. I'm not sure what they need since I haven't talked to the client. I would defer to @akahure. This may also have to do with budget/timeline for this, which I'm also not sure about.

Here is an image of what the double progress bars look like: 
![summary splash](https://user-images.githubusercontent.com/1584163/79935704-9236ad00-8423-11ea-971b-d42e6e05ecef.png) based on the spreadsheet.

**Comment by akahure** [comment]
@rowo we received final feedback comments for the splash page once it is loaded on the site 
- The cards should only showcase the main title (category) as below, and only the core indicator (max and min) shown. 
![Screenshot - 2020-05-05T120851 825](https://user-images.githubusercontent.com/43808025/81050747-36f7c680-8ec9-11ea-802c-7bb9c17e5340.png)
- Request to have all the Indicator cards and core indicators appear on a single page view, arranged horizontally
I have updated the spreadsheet below, it has the title and the core indicator highlighted in a light grey as below 
![Screenshot - 2020-05-05T123736 593](https://user-images.githubusercontent.com/43808025/81053140-4d078600-8ecd-11ea-97ca-84325018f039.png)

file: https://docs.google.com/spreadsheets/d/1zw2ek1vjMlaVWaWKqpMGxrmd5ugETZEhp4wjESg605o/edit#gid=0

**Comment by rowo** [comment]
@akahure they want it to horizontally scroll? Sideways?

**Comment by akahure** [comment]
@rowo something like the below mockup. The numbering to show the order of how it should appear. 

![Blank Diagram - E-Commerce Category Page Basic Wireframe](https://user-images.githubusercontent.com/43808025/81087771-c66d9b80-8f02-11ea-865f-ee09423b6a6a.png)

**Comment by rowo** [comment]
@akahure Is this about correct?

![summary splash copy](https://user-images.githubusercontent.com/1584163/81368330-80a90280-90bd-11ea-9970-221ccd4a1438.png)

**Comment by akahure** [comment]
@rowo this is right it should be like this. Considering there are 10 core categories (for indicators), we will need to include 10 cards, we will adjust the cards to have 4 per row. One key client requirement is to have all the cards appear on the same page (without scrolling), do you have any reservations?

**Comment by ukanga** [comment]
@akahure Should we close this issue? Is this still out standing?

**Comment by akahure** [comment]
> @akahure Should we close this issue? Is this still outstanding?

Hi @ukanga its currently on hold until the client confirms we can proceed. I will flag it on hold but not close it
